pkg/k8s/portforward.go

```diff
@@ -162,6 +163,33 @@ func (pf *PortForward) Run() error {
 	return fw.ForwardPorts()
 }
 
+// Init creates and runs a port-forward connection
+func (pf *PortForward) Init(wait ...chan struct{}) error {
+	log.Debug("Starting port forward")
+
+	failure := make(chan error, 1)
+
+	go func() {
+		if err := pf.Run(); err != nil {
+			failure <- err
+		}
+		if len(wait) > 0 {
+			close(wait[0])
```

Comment:
Why do we only close the first wait channel?

Reply:
@adleong the plan was to have `Init` receive one or no args. Anyways, I'll use @siggy's suggestion to leverage the `stopCh` channel instead.